azerothcore/azerothcore-wotlk
# (DB) Work Work Audio Broadcast fix

Although the Audio is Loud, It only played from the NPC when the Player with in 20 yards when the lazy peon starts the cutting the tree animation.  Currently AZ and TC has it playing from the player with no range limit. This is NOT CORRECT, and has been wrong in many video recordings that are apparrently from Private servers and not from retail official. However tc seems to have other wizardly invovled with their audio playing with this case in particular that is not clear to me as it is ignoring their own action para 2, but that is not a issue here.
My changes are based on confirm sniff verifications. This should solve the insanity of WORK WORK being spammed none stop at times in the area. Sniff used was based on TBC Classic, There was no changes to zone\npc\or quest related between tbc and wotlk.

## Changes Proposed:
- Makes quest Lazy Peon audio broadcast work work only from themselves and not the player (action_para2)
- Sets it up where the target_type is the player and only triggers with in 20 yards

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4303

## SOURCE:

-- tbc sniff submission on the sniff collective https://github.com/MDic-s-Preservation-Archives/Sniff-Collective/commit/8b267d2444c805693d2254c9bc522a21ba6bf5ab

## Tests Performed:

- Builds
- Runs

## How to Test the Changes:


1. goto valley of trial and be near a lazy peon
2. when they stand up and walk to the tree they will play the audio if within 20 yards of them
3. stand beyond 20 yards from them (click and highlight them and type .dist to confirm distance) and wait and observer until they go back to the tree
4. When they start chopping the tree, you will not hear the audio WORK WORK


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
